packages/integrations/node/README.md

```diff
@@ -109,6 +109,16 @@ node ./dist/server/entry.mjs
 
 For standalone mode the server handles file servering in addition to the page and API routes.
 
+
+#### Runtime HOST and PORT
+
+It is possible to override built HOST and PORT values with [runtime environment variables](https://docs.astro.
+build/en/guides/environment-variables/#using-the-cli)
```

Comment:
Thanks @Scttpr! I think this is a good place for this.

Made a couple of tweaks here. I don’t think we need all-caps when referring to the concept of host/port. I’m also not sure the link is that helpful as that section of the docs describes running the Astro dev server. How does this look?

```suggestion
#### Custom host and port

You can override the host and port the standalone server runs on by passing them as environment variables at runtime:
```

Reply:
You are totally right!